File: app/src/main/java/com/artmurka/artmurkaapp/model/pojo/itemlist/wishList/EntryPhoto.java

package com.artmurka.artmurkaapp.model.pojo.itemlist.wishList;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class EntryPhoto {


    @SerializedName("num_photos")
    @Expose
    private Long numPhotos;
    @SerializedName("def_photo")
    @Expose
    private DefPhoto defPhoto;

    public Long getNumPhotos() {
        return numPhotos;
    }

    public void setNumPhotos(Long numPhotos) {
        this.numPhotos = numPhotos;
    }

    public DefPhoto getDefPhoto() {
        return defPhoto;
    }

    public void setDefPhoto(DefPhoto defPhoto) {
        this.defPhoto = defPhoto;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/wishList/EntryPhoto.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.wishList

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class EntryPhoto {


    @SerializedName("num_photos")
    @Expose
    var numPhotos: Long? = null
    @SerializedName("def_photo")
    @Expose
    var defPhoto: DefPhoto? = null

}
